File: flight-tracking-service/src/main/java/github/wozniak/flighttrackingservice/dto/AirportDTO.java
package github.wozniak.flighttrackingservice.dto;

import github.wozniak.flighttrackingservice.entity.Airport;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
public class AirportDTO {

    private String icaoCode;
    private String name;
    private double latitude;
    private double longitude;
    private String continent;
    private String country;
}

File: backend/src/Domain/airport/Dto/AirportDto.cs
using backend.Domain.airport.Entity;

namespace backend.Domain.airport.Dto;

public class AirportDto
{
    public string AirportCode { get; set; }
    public string AirportName { get; set; }
    public double Latitude { get; set; }
    public double Longitude { get; set; }
    public string City { get; set; }
    public string State { get; set; }
    public string Country { get; set; }
    public double EconomicValuePercentile { get; set; }
    public string MarketType { get; set; }

    public AirportDto(Airport airport)
    {
        AirportCode = airport.AirportCode;
        AirportName = airport.AirportName;
        Latitude = airport.Latitude;
        Longitude = airport.Longitude;
        City = airport.City;
        State = airport.State;
        Country = airport.Country;
        EconomicValuePercentile = airport.EconomicValuePercentile;
        MarketType = airport.MarketType.ToString();
    }
}